src/browser.js

```diff
@@ -1,8 +1,19 @@
+/** Copyright (c) 2018 Uber Technologies, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
 import React from 'react';
 import {createPlugin} from 'fusion-core';
 import {HelmetProvider} from 'react-helmet-async';
 
-export default __BROWSER__ &&
+import type {FusionPlugin} from 'fusion-core';
+
+const plugin =
+  __BROWSER__ &&
```

Comment:
Hmm, @ganemone - did you find that this broke treeshaking?

Reply:
Oh, thought this is what we did to fix treeshaking? Let's find the previous example of where it broke.

Reply:
I believe this was the tree-shaking fix you're referring to: https://github.com/fusionjs/fusion-plugin-rpc/commit/92c3a8ae4c53071bf31dfe1343c5901c75da7c6e

It looks like this is the correct way of doing things, and the incorrect way is:
```
export default ((__NODE__ && plugin: any): RPCPluginType);
```

Reply:
Got it, thanks for hunting that down!